index.js

```diff
@@ -64,12 +77,38 @@ module.exports = function(options) {
       allowNamespaces: options.allowNamespaces !== false,
       allowNamespacedObjects: !!options.allowNamespacedObjects
     }, Parser);
-  }
+  };
 };
-module.exports.tokTypes = tok;
+
+// This is `tokTypes` of the peer dep.
+// This can be different instances from the actual `tokTypes` this plugin uses.
+Object.defineProperty(module.exports, "tokTypes", {
+  get: function get_tokTypes() {
+    return getJsxTokens(require("acorn")).tokTypes;
+  },
+  configurable: true,
+  enumerable: true
+});
 
 function plugin(options, Parser) {
+  const acorn = Parser.acorn || require("acorn");
+  const acornJsx = getJsxTokens(acorn);
+  const tt = acorn.tokTypes;
+  const tok = acornJsx.tokTypes;
```

Comment:
Would it make sense to put these as a static property (say, `jsxTokenTypes`) on the returned class? The fact that they were exported before means that people probably use them, and with these changes the top `tokTypes` export becomes a bit useless. I think it makes sense to leave it (with maybe a deprecation note in the docs), but we should also provide a more robust alternative.

Reply:
I have put those as a static property already. Would you request renaming it?

Reply:
Currently, users can access those via `Parser.acornJsx.tokTypes`. It's similar to `Parser.acorn.tokTypes`.

As PR description mentioned, [Espree will use the `JsxParser.acornJsx`](https://github.com/eslint/espree/pull/426/files?w=1) to modify tokens.

Reply:
Ah, I missed that, sorry.